packer.json.dev

```diff
@@ -0,0 +1,48 @@
+{
+  "builders": [
+    {
+      "type": "amazon-ebs",
+      "name": "empire-us-east-1",
+      "access_key": "",
+      "secret_key": "",
```

Comment:
Actually don't need that - packer already looks for those environment variables if the variables in the template aren't set:

https://www.packer.io/docs/builders/amazon.html#specifying-amazon-credentials